instrumentation/spring/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -273,6 +273,12 @@
       "description": "Enable the <code>@WithSpan</code> annotation.",
       "defaultValue": true
     },
+    {
+      "name": "otel.instrumentation.common.db-statement-sanitizer.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Enables the DB statement sanitization for R2DBC.",
+      "defaultValue": true
```

Comment:
it is weird that this applies only to r2dbc, I'd assume that this would also apply to jdbc. I'd make the description more generic.

Reply:
no, this feature is missing for JDBC currently - but I'm happy to add that in a follow up PR :smile: